travis_specific/install_ganache.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 
 npm install -g npm
-npm install
```

Comment:
npm appears to already be installed (https://docs.travis-ci.com/user/languages/javascript-with-nodejs/); do we still need the line above?